feature-libs/quote/components/links/quote-links.component.html

```diff
@@ -20,6 +20,26 @@
           {{ 'quote.links.quotes' | cxTranslate }}
         </a>
       </li>
+      <li
+        *ngIf="
```

Comment:
@suprishi, the condition should be moved from button HTML element to li element, otherwise `|` is displayed when `Download proposal` button is not visible.